hilti/toolchain/src/compiler/jit.cc

```diff
@@ -194,6 +182,45 @@ JIT::~JIT() {
 }
 
 hilti::Result<std::shared_ptr<const Library>> JIT::build() {
+    auto _guard = util::scope_exit([&]() {
+        // Point out user diagnostics file if it exists.
+        if ( ! hilti::rt::filesystem::exists(UserDiagnosticsFile) )
+            return;
+
+        std::cerr << util::fmt(R"(
+=== Uh oh ... C++ compilation failed ===
+
+This is unexpected and should not happen. Please help us fix the issue
+by opening a ticket at https://github.com/zeek/spicy/issues/new.
+
+Please include the content of the following file with your report:
+
+  %s
+
+That file contains diagnostics from the C++ compiler. Please also
+include your Spicy code that triggered this error, ideally reduced to
+a small snippet still exhibiting the issue if you can.
+
+===
+
+)",
+                               rt::filesystem::absolute(UserDiagnosticsFile));
+
+        if ( auto* show_output = getenv("HILTI_JIT_SHOW_CXX_OUTPUT"); show_output && *show_output ) {
```

Comment:
This is a matter of taste, but since this makes no distinction between the values, I'd prefer this toggle just check for the presence of the env var rather than having to set it to some random value. This way `HILTI_JIT_SHOW_CXX_OUTPUT= spicyc -j test.spicy` works. Happy either way though

```suggestion
        if ( hilti::rt::getenv("HILTI_JIT_SHOW_CXX_OUTPUT").has_value() ) {
```

Reply:
Actually I don't like it working without any value at all, and we're using the same pattern elsewhere already, so prefer to leave as is.